nn_sdwan/assets/monitors/high-latency-monitor.json

```
@@ -0,0 +1,23 @@
+{
+    "id": 65951821,
+    "name": "High Latency Alert [>100ms]",
+    "type": "query alert",
+    "query": "avg(last_5m):avg:nn_sdwan.app_aware_routing.latency{*} > 100",
+    "message": "A link has detected  high latency.",
+    "tags": [],
```

Should have a tag in the form of `integration:<integration_name>`. Example:
```suggestion
    "tags": [
      "integration:nn_sdwan"
    ],
```